src/jobservice/job/tracker.go

```diff
@@ -232,13 +232,14 @@ func (bt *basicTracker) CheckIn(message string) error {
 
 	bt.refresh(current, message)
 	err := bt.fireHookEvent(current, message)
```

Comment:
The consideration here is the status hook event sending error should not block the following status update because hook failure can be repaired by the reaper process which depends on the status data updated here.

@alrs and also cc @wy65701436